test/registrationTest.js

```diff
@@ -6,39 +6,50 @@ chai.use(require('chai-http'));
 
 const app = require('../index.js'); // Our app
 
-var body = {
-    first:"test",
-    last:"test1",
-    username: "fakeuser",
-    email:"test@fake.email.com",
-    password:"test"
+var registrationBody = {
+    "first":"test",
+    "last":"test",
+    "email":"test@fake.email.com",
+    "password":"test"
 };
 
-describe('Registration endpoint', function () {
-
-    this.timeout(5000); // How long to wait for a response (ms)
-
-    before(function () {
-
-    }
-    );
-
-    after(function () {
+var signinBody = {
+    "username":"test",
+    "password":"password"
+}
 
-    }
-    );
+describe('Home page endpoint', function () {
+    it('should load the home page', function() {
+        return chai.request(app)
+        .get('/')
+        .then(function(res) {
+            expect(res).to.have.status(200);
+        })
+    })
+})
 
+describe('Registration endpoint', function () {
     // POST register fake account.
```

Comment:
I changed this test to only test is a 200 response is received and not check what is returned